### PR TITLE
add extra logs to investigate lock type issues

### PR DESCRIPTION
**Pull request by mgallien**

### CODE REVIEW

File: src/gui/folder.cpp [hunk 1]
```diff
@@ -649,12 +649,15 @@ void Folder::slotFilesLockReleased(const QSet<QString> &files)
...
         const auto canUnlockFile = isFileRecordValid
             && rec._lockstate._locked
-            && rec._lockstate._lockOwnerType == static_cast<qint64>(SyncFileItem::LockOwnerType::TokenLock)
+            && (!_accountState->account()->capabilities().filesLockTypeAvailable() || rec._lockstate._lockOwnerType == static_cast<qint64>(SyncFileItem::LockOwnerType::TokenLock))
             && rec._lockstate._lockOwnerId == _accountState->account()->davUser();
 
         if (!canUnlockFile) {
-            qCDebug(lcFolder) << "Skipping file" << file << "with rec.isValid():" << rec.isValid()
-                             << "and rec._lockstate._lockOwnerId:" << rec._lockstate._lockOwnerId << "and davUser:" << _accountState->account()->davUser();
+            qCDebug(lcFolder) << "Skipping file" << file
```
Comment (allexzander) [inline src/gui/folder.cpp:656]: @mgallien not sure if you don't want to change this to `qCInfo/qCWarning` now. Debug logs are not visible in Relese.

Comment (mgallien) [reply src/gui/folder.cpp]: done

**Review by allexzander — APPROVED** [review]

**Pull request merged**